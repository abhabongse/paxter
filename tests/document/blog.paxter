@python##"
    def foo():
        pass
"##

@h1{My Blog!}

Hello, World!

@h2{Source Code}

Please visit @link["https://github.com"]{GitHub} to see
the @bold{source} @bold{code} for this project.

@numbered_list[
    {This is the first item.},
    {This is the second item.},
    {This is the third item.},
    {
        This is the @italic{last} item, first paragraph.

        This is another paragraph within an item.

        @blockquote{No way!}
    },
]

@blockquote{
    @bulleted_list[
        ##{Left {word}}##,
        ##{Right {word}}##,
        "Another word",
    ]

    Cool!
}

@blockquote{
    This is one paragraph.
}

@blockquote{
    This is @italic{the} first paragraph.

    This is @italic{the} second paragraph.
}

@hrule

@h2{Do you know?}

Do you know that 2 > 1?

@paragraph{@code{This paragraph is entirely in monospaced.}}

See the lovely @uline{image} below.

@image["images/hello.png", alt="hello image"]

@foo[]
