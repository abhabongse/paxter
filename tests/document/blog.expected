<h1>My Blog!</h1><p>Hello, World!</p><h2>Source Code</h2><p>Please visit <a href="https://github.com">GitHub</a> to see
the <b>source</b> <b>code</b> for this project.</p><ol><li>This is the first item.</li><li>This is the second item.</li><li>This is the third item.</li><li><p>This is the <i>last</i> item, first paragraph.</p><p>This is another paragraph within an item.</p><blockquote>No way!</blockquote></li></ol><blockquote><ul><li>Left {word}</li><li>Right {word}</li><li>Another word</li></ul><p>Cool!</p></blockquote><blockquote>This is one paragraph.</blockquote><blockquote><p>This is <i>the</i> first paragraph.</p><p>This is <i>the</i> second paragraph.</p></blockquote><hr /><h2>Do you know?</h2><p>Do you know that 2 &gt; 1?</p><p><code>This paragraph is entirely in monospaced.</code></p><p>See the lovely <u>image</u> below.</p><img src="images/hello.png" alt="hello image" />